InventoryManagementSystem/InventoryManagementSystem/InventoryManagementExceptions.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace InventoryManagementSystem
+{
```

Comment:
can add more custom exceptions

Reply:
done

Reply:
What is the difference between **InvalidQuantityException** and **InvalidDataException**, as for quantity validation also you can and you are using the InvalidDataException only. [Reference Line no: 74, ProductManager.cs]

Reply:
you're right, there is no real difference between them, so I am removing **InvalidQuantityException**.